doc/btp_tmap.txt

```diff
@@ -0,0 +1,46 @@
+TMAP Service (ID 27)
```

Comment:
Two more occurrences of TMAP Service ID needs update, here
```suggestion
TMAP Service (ID 28)
```
and a missing entry `28 TMAP Service` in doc/overview.txt